tensorflow_serving/model_servers/main.cc

```diff
@@ -200,7 +200,8 @@ void RunServer(int port, std::unique_ptr<ServerCore> core,
   std::shared_ptr<grpc::ServerCredentials> creds = InsecureServerCredentials();
   builder.AddListeningPort(server_address, creds);
   builder.RegisterService(&service);
-  builder.SetMaxMessageSize(tensorflow::kint32max);
+  // Raise max frame size for larger request if it is reasonable
+  builder.SetMaxMessageSize(104857600);
```

Comment:
Just to confirm, are you running with TensorFlow Serving at head and are running into a problem that the requests are too large? I think this PR should not be necessary because the current limit is greater than the newly proposed 100M. When we added this change about a month ago in [this commit](https://github.com/tensorflow/serving/commit/543c89517cdcf944e79b3cee97aed0fcb1679a16), we tested with requests that were in the order of 100M and it worked (and of course failed with the old default of 4M).